### PR TITLE
Release 6.8.7  -- Minor tweaks

### PR DESCRIPTION
**Pull request by briskt**
### Changed
- Wait to print summary message in case more log events are queued.
- Use range int syntax
- Use `slices.Contains` in place of home-built functions

### Fixed
- Correct minor errors in a comment about ExtraMember config

### CODE REVIEW

File: google/groups.go [hunk 1]
```diff
@@ -111,14 +112,10 @@ func (g *GoogleGroups) ListUsers(desiredAttrs []string) ([]internal.Person, erro
...
-			continue
-		}
-		if isExtraOwner, _ := internal.InArray(nextMember.Email, g.GroupSyncSet.ExtraOwners); isExtraOwner {
-			continue
-		}
-		if isExtraMember, _ := internal.InArray(nextMember.Email, g.GroupSyncSet.ExtraMembers); isExtraMember {
+		// Do not include ExtraManager, ExtraOwners, or ExtraMember in list to prevent inclusion in delete list
+		if slices.Contains(g.GroupSyncSet.ExtraManagers, nextMember.Email) ||
+			slices.Contains(g.GroupSyncSet.ExtraOwners, nextMember.Email) ||
+			slices.Contains(g.GroupSyncSet.ExtraMembers, nextMember.Email) {
```
Comment (mtompset) [inline google/groups.go:118]: Nice optimization.

File: internal/internal.go [hunk 2]
```diff
@@ -224,17 +223,17 @@ func RunSyncSet(logger *log.Logger, source Source, destination Destination, conf
 
 	results := destination.ApplyChangeSet(changeSet, eventLog)
 
-	logger.Printf("Sync results: %v users added, %v users updated, %v users removed\n",
-		results.Created, results.Updated, results.Deleted)
-
-	for i := 0; i < 100; i++ {
+	for range 100 {
```
Comment (mtompset) [inline internal/internal.go:226]: But the for loop style was more readable for those less familiar with go. :)

**Review by mtompset — APPROVED** [review]

**Review by jason-jackson — APPROVED** [review]

File: internal/internal.go (hunk 2 above)
Comment (briskt) [reply internal/internal.go:226]: You have a point. Makes me wonder how many different styles of `for` loop exist among major programming languages. 🤔

File: google/groups.go (hunk 1 above)
Comment (briskt) [reply google/groups.go:118]: The new `slices` package is a nice addition to the standard library.

**Pull request merged**